app/models/category.rb

```diff
@@ -210,7 +211,7 @@ def related_attribute_friendly_ids=(ids)
   def as_json_for_data
     {
       "id" => id,
-      "name" => name,
+      "name" => self[:name], # avoid localization
```

Comment:
I assume this means there's wip for changing localizations or something? Unless I'm mistaken, all the scripts using this only work for the english locale anyways

Reply:
Nothing WIP on my side; just finding spots where we need to turn it off — keeping that targeted for the moment until we see more (but adding a comment so folks understand why I'm not using the method, since it'll be surprising to a casual reader)